src/roman-numerals/index.js

```diff
@@ -4,6 +4,32 @@
  * @param {string} roman The all-caps Roman numeral between 1 and 3999 (inclusive).
  * @returns {number} The decimal equivalent.
  */
-function romanToDecimal(roman) {}
+function romanToDecimal(roman) {
+    let decimal = 0;
+    let romanObj = {                                     //roman to decimal conversions
+        i: 1,
+        v: 5,
+        x: 10,
+        l: 50,
+        c: 100,
+        d: 500,
+        m: 1000
+    }
+    let romanArr = roman.toLowerCase().split('');      //convert input string to an array of lower case letters
```

Comment:
There was no need to do this if you had just made created the keys in `romanObj` in uppercase.
The key is to do less work.